### PR TITLE
3.1.0 release

### PR DESCRIPTION
**Pull request by Grummfy**

### CODE REVIEW

File: _posts/2017-07-18-release-3-1-0.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* 277 files changed,
+* 7 contributors,
+* 7 [new features][changelog],
+* 1 [bug fix][changelog].
+
+## What's new
+
+Several new feature and a bug fix came with this release.
+
+Thing related to PHP version 7.1 & 7.2 to let atoum be ready for the next version of PHP:
```
Comment (mikaelrandy) [inline _posts/2017-07-18-release-3-1-0.markdown]: Things*
PHP next version*

File: _posts/2017-07-18-release-3-1-0.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* The mock generator can now manage nullable type and handle the void type
+* The mock generator support strict types
+* The object asserter has been renamed phpObject to avoid issue related to reserved words in PHP 7.2
```
Comment (mikaelrandy) [inline _posts/2017-07-18-release-3-1-0.markdown]: renamed to*

File: _posts/2017-07-18-release-3-1-0.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Thing related to PHP version 7.1 & 7.2 to let atoum be ready for the next version of PHP:
+* The mock generator can now manage nullable type and handle the void type
+* The mock generator support strict types
+* The object asserter has been renamed phpObject to avoid issue related to reserved words in PHP 7.2 
+
+### Assertions & asserters
+
+#### Exception
+Now the asserter on exception has an `isInstanceOf` that can be used assertions without brackets.
```
Comment (mikaelrandy) [inline _posts/2017-07-18-release-3-1-0.markdown:32]: Maybe give an little exemple

File: _posts/2017-07-18-release-3-1-0.markdown [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+#### Stream
+Now the asserter on stream has a `isRead` and `isWritten`  that can be used assertions without brackets.
```
Comment (mikaelrandy) [inline _posts/2017-07-18-release-3-1-0.markdown:35]: Same

File: _posts/2017-07-18-release-3-1-0.markdown (hunk 3 above)
Comment (Grummfy) [reply _posts/2017-07-18-release-3-1-0.markdown:32]: in fact I don't really see the inntrest of using isInstanceOf without brackets ... I just copy/paste the changelog, but this make no sense...

File: _posts/2017-07-18-release-3-1-0.markdown (hunk 4 above)
Comment (Grummfy) [reply _posts/2017-07-18-release-3-1-0.markdown:35]: done ;)

File: _posts/2017-07-18-release-3-1-0.markdown (hunk 3 above)
Comment (Grummfy) [reply _posts/2017-07-18-release-3-1-0.markdown:32]: @Hywan or @guiled can you came back on this one?

Comment (Hywan) [reply _posts/2017-07-18-release-3-1-0.markdown:32]: @guiled Probably :-)

**Review by Hywan — APPROVED** [review]

**Comment by Grummfy** [comment]
So, I got no news, I suppose we should publish it?

**Pull request merged**